jerry-core/parser/js/js-parser.c

```diff
@@ -62,6 +62,12 @@ parser_compute_indicies (parser_context_t *context_p, /**< context */
   uint16_t ident_count = 0;
   uint16_t const_literal_count = 0;
 
+#if ENABLED (JERRY_RESOURCE_NAME)
+  /* Resource name will be stored as the last const literal. */
+  const_literal_count++;
+  context_p->literal_count++;
```

Comment:
There is a limit for literals (32K or something), perhaps that check should be added here.